src/components/footer/index.tsx

```diff
@@ -107,13 +121,15 @@ function Footer({
 						innerElement = (
 							// Note: we do follow this rule, eslint just doesn't recognize it
 							// eslint-disable-next-line react/jsx-no-target-blank
+
 							<a
 								className={s.linkAction}
 								href={item.href}
 								target={item.opensInNewTab ? '_blank' : undefined}
 								rel={item.opensInNewTab ? 'noreferrer' : undefined}
 							>
 								{textElement}
+								{item.opensInNewTab ? <IconExternalLink16 /> : null}
 							</a>
```

Comment:
📌 I think we probably want to leverage `components/Link` here since it handles making sure "opens in new tab" is communicated to screen reader users as well?

Reply:
Just updated!